index.html

```diff
@@ -53,6 +53,140 @@ <h2>
+                         </div>
+                         <div class="open"><a href="#">See Project</a></div>
+                    </div>
+                </article>
+                <article class="card">
+                    <div class="sane">
+                    </div>
+                    <div class="content">
+                        <h1>Multi-post Stories <br>Gain + Glory</h1>
+                        <div class="category">
+                                <div class="category1"><a href="#">Ruby on Rails</a></div>
+                                <div class="category2"><a href="#">Css</a></div>
+                                <div class="category3"><a href="#">Javascript</a></div>
+                                <div class="category4"><a href="#">Html</a></div>
+                         </div>
+                         <div class="open"><a href="#">See Project</a></div>
+                    </div>
+                </article>
+            </div>
+        </section>
+        <section class="about">
+            <h1>About</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="resume"><a href="#">Get My Resume</a></div>
+        </section>
+        <section class="skill">
+            <div class="weight">
+                <div class="featured"><img src="./image/icon_lang.png" alt=""></div>
+                <h1>Languages</h1>
+                <div class="cores"><ul>
+                    <li><a href="#">Javascript</a></li>
+                    <li><a href="#">Ruby</a></li>
+                    <li><a href="#">Html</a></li>
+                    <li><a href="#">Css</a></li>
+                </ul>
+            </div>
+            </div>
+            <div class="weight">
+                <div class="featured"><img src="./image/icon-frameworks.png" alt=""></div>
+                <h1>Frameworks</h1>
+                <div class="cores"><ul>
+                    <li><a href="#">Bootstrap</a></li>
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">RSpec</a></li>
+                    <li><a href="#">Capybara</a></li>
+                    <li><a href="#">Selenium</a></li>
+                </ul>
+            </div>
+            </div>
+            <div class="weight">
+                <div class="featured"><img src="./image/icon_skills.png" alt=""></div>
+                <h1>Skills</h1>
+                <div class="cores"><ul>
+                    <li><a href="#">Codekit</a></li>
+                    <li><a href="#">GitHub</a></li>
+                    <li><a href="#">Codepen</a></li>
+                    <li><a href="#">Github</a></li>
+                    <li><a href="#">Terminal</a></li>
+                </ul>
+            </div>
```

Comment:
**OPTIONAL**
- It would be better if instead of using two separate sections for the "About Myself" section you can use only one section for the whole of it since the technology categories are also part of that section.

**REQUIRED**
- The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li. Please kindly fix that.